website/docs/guides/container_service_extension_4_x_cluster_management.html.markdown

```diff
@@ -36,47 +36,49 @@ section to understand how it works.
 this RDE is named `k8s_cluster_instance`. The important arguments to take into account are:
 
-- **`resolve` must be always `false`**, because the CSE Server is responsible for performing the RDE resolution when the
+* **`resolve` must be always `false`**, because the CSE Server is responsible for performing the RDE resolution when the
   TKGm cluster is completely provisioned, so Terraform should not interfere with this process.
-- **`resolve_on_removal` must be always `true`**, because the RDE is resolved by the CSE Server and not by Terraform. If one
+* **`resolve_on_removal` must be always `true`**, because the RDE is resolved by the CSE Server and not by Terraform. If one
   wants to execute a Terraform destroy without the RDE being resolved, the operation will fail. Being `true` assures that Terraform
   can perform a Terraform destroy in every case. See ["Deleting a Kubernetes cluster"](#deleting-a-kubernetes-cluster) section for more info.
 
 The [`vcd_rde`][rde] argument `input_entity` is taking the output of the Terraform built-in function `templatefile`, that references
 a JSON template that is located at [here][tkgmcluster_template]. This function will set the correct values to the following
 placeholders that can be found in that file:
 
-- `vcd_url`: The VCD URL, the same that was used during CSE installation.
-- `name`: This will be the TKGm cluster name. It must contain only lowercase alphanumeric characters or '-',
+* `vcd_url`: The VCD URL, the same that was used during CSE installation.
+* `name`: This will be the TKGm cluster name. It must contain only lowercase alphanumeric characters or '-',
   start with an alphabetic character, end with an alphanumeric, and contain at most 31 characters.
-- `org`: The Organization in which the TKGm clusters will be created. In this guide it was created as `tenant_org` and named
+* `org`: The Organization in which the TKGm clusters will be created. In this guide it was created as `tenant_org` and named
   "Tenant Organization" during CSE installation phase.
-- `vdc`: The VDC in which the TKGm clusters will be created. In this guide it was created as `tenant_vdc` and named
+* `vdc`: The VDC in which the TKGm clusters will be created. In this guide it was created as `tenant_vdc` and named
   "Tenant VDC" during CSE installation phase.
-- `capi_yaml`: This must be set with a valid single-lined CAPVCD YAML, that will be explained next.
-- `delete`: This is used to delete a cluster. See ["Deleting a Kubernetes cluster"](#deleting-a-kubernetes-cluster) section for more info.
+* `api_token`: The API token that corresponds to the user that will create the cluster. This is set automatically with a reference to
+  a `vcd_api_token` resource. One can customise the path to the JSON file where the API token is stored using `cluster_author_token_file` variable.
+* `capi_yaml`: This must be set with a valid single-lined CAPVCD YAML, that will be explained next.
```

Comment:
> This must be set with a valid single-lined CAPVCD YAML, **that will be explained next.**

Next where? :) Better for us to give a specific context-less link.

Reply:
Changed to `, that is explained below.`, I can also put `, that is explained a few paragraphs below.`

Reply:
```
* `capi_yaml`: This must be set with a valid single-lined CAPVCD YAML, that is explained below.
```

Thanks! Just noticed "single-line**d**" - what is the "d" for? :)

Reply:
Probably a misspelling on my side 😄 , just wanted to make a graphical adjective out of it.

Reply:
I rephrased this one with a more clear sentence.